modules/core/keeper/msg_server.go

```diff
@@ -856,19 +856,22 @@ func (k Keeper) ChannelUpgradeAck(goCtx context.Context, msg *channeltypes.MsgCh
 
 	app, ok := k.Router.GetRoute(module)
 	if !ok {
-		ctx.Logger().Error("channel upgrade ack failed", "port-id", msg.PortId, "error", errorsmod.Wrapf(porttypes.ErrInvalidRoute, "route not found to module: %s", module))
-		return nil, errorsmod.Wrapf(porttypes.ErrInvalidRoute, "route not found to module: %s", module)
+		err = errorsmod.Wrapf(porttypes.ErrInvalidRoute, "route not found to module: %s", module)
```

Comment:
we could reinitialise new errors here as they'd be within the scope of the `if`, but I don't really feel super strong on it and happy to reuse `err`

Reply:
will push it in as is for now, can't say I have a strong pref either way too (slight preference for yours since it scopes values nicely but not enough pref to make the change)